### ISSUE TITLE
Is there any way to remove any Google related text in the browser?

### ISSUE THREAD

**Issue by David-Frick** [Eloston/ungoogled-chromium#900]
I really like this project. I use it on Arch Linux, will probably get it on my Debian distro, and will suggest it to families and friends! I was wondering if there is a way to edit out the text mentioning "Google" in settings. I notice it talks about Google sync, and mentions Google elsewhere in there.

To me a solution, is having no mention of Google. Also, is there a way to get Ublock origins or other extensions with this (preferably not through the Google app store)

Side note: I noticed you guys are looking for a Debian maintainer. I am trying to help maintain another package for Debian. I will probably check back in and help with this package if yall still need help. First, I will learn how it works (Debian's documentation on packaging.maintaining vs Arch Linux is night and day, not in Debian's favor...)

Thanks for this great package,
David

**Comment by jstkdng** [comment]
Hello David, let me answer your questions.
1. yes, it could be possible, but why?
2. you can download the .crx file from the developers themselves and install it directly, that makes it hard to update them though.
3. really? I thought Eloston was maintaining the Debian repo, oh well.

**Comment by David-Frick** [comment]
Let me check on the debian repo. I did not see it listed online, the debian website is a bit chaotic! So that might be me.

I definitely want to be able to update it!

I do not know, just seeing google even in the settings puts me on edge :/

-------- Original Message --------

**Comment by Eloston** [comment]
> Side note: I noticed you guys are looking for a Debian maintainer. I am trying to help maintain another package for Debian. I will probably check back in and help with this package if yall still need help. First, I will learn how it works (Debian's documentation on packaging.maintaining vs Arch Linux is night and day, not in Debian's favor...)

Yes! It would be awesome if ungoogled-chromium is included in Debian. Right now we are working towards getting ungoogled-chromium built and published in a PPA, so we're sorting out some issues along the way. However, we're not very familiar with Debian packaging guidelines so we'd appreciate help in figuring out what we need to do for that.

> really? I thought Eloston was maintaining the Debian repo, oh well.

Well if someone wants to help or take over maintenance of ungoogled-chromium-debian's `debian_buster` branch, that'd be nice. I want to spend more time on the longer-term issues with larger impact.

**Comment by Eloston** [comment]
Just to be clear @David-Frick is talking about ungoogled-chromium inclusion in Debian (which we should discuss more in #27). @jstkdng is talking about our Debian packages published on GitHub along with all other binaries.

> I do not know, just seeing google even in the settings puts me on edge :/

In that cse, you'll be estatic to see Google's license header at the top of every source file ;)

All jokes aside, removing the switches could be helpful to eliminate any confusion/doubts about whether those flags will do anything (because they won't). Since this is a cosmetic change of sorts, it isn't a high priority for most of us (who are more technical), but we'd be happy to merge a PR.

**Comment by David-Frick** [comment]
Okay, cool! I will keep in touch! I am working on getting in contract with a Debian mentor to hammer out any packaging questions I have.

-------- Original Message --------

**Comment by jstkdng** [comment]
>Well if someone wants to help or take over maintenance of ungoogled-chromium-debian's debian_buster branch, that'd be nice. I want to spend more time on the longer-term issues with larger impact.

I could help with maintenance of that branch since I'm using ubuntu for work right now, the process should be basically the same right?

**Comment by David-Frick** [comment]
I do not know. I am looking into debian packaging now, but their website is hard to decipher documentation wise. Could you walk me through how ubuntu maintains and submits packages? I would assume at the very least that Ubuntu provides the packages for Ubuntu and its flavors, Linux Mint, KDE neon and the other Ubuntu-based distros.

**Comment by jstkdng** [comment]
I don't know about how ubuntu creates its packages yet, but I think they just create the .deb files, upload them to their main servers and update the package database. After that, the secondary servers just mirror the packages from the main servers.
Submission is only made by trusted users, so I doubt any of us could submit ungoogled-chromium to the official repositories. The least we could do is create a PPA on launchpad, and maybe contact a trusted user to maintain ungoogled-chromium for us.
Regarding the last part, Ubuntu only provides packages to the flavor it oficially supports. Mint and KDE neon aren't one of them afaik. They can either use those packages or maintain their own.

**Comment by Frick-David** [comment]
Do Debian packages get sent downstream ie a Ubunyu User can ger Debian packages from their repository?

-------- Original Message --------

**Comment by jstkdng** [comment]
Nope, as debian packages could have different dependency versions and or 
dependencies that don't exist on ubuntu.

**Comment by Frick-David** [comment]
Gotcha. That makes sense!

-------- Original Message --------

**Comment by Eloston** [comment]
> I could help with maintenance of that branch since I'm using ubuntu for work right now, the process should be basically the same right?

The process should be documented in the README in the debian_buster branch. If the instructions aren't clear, let me know.

> Do Debian packages get sent downstream ie a Ubunyu User can ger Debian packages from their repository?

Technically if we get ungoogled-chromium packaged for Debian, Ubuntu should eventually get the package in their repos.
